docs/SemanticFileSystemGettingStarted.md

```diff
@@ -0,0 +1,130 @@
+Semantic File System GettingStarted
+================================================
+
+Contents
+--------
+
+*   [1 Prerequisites](#Prerequisites)
+*   [2 Creating a Semantic project](#Creating-a-Semantic-project)
+*   [3 A short walk-through: some places to know about when working with the Semantic File System](#A-short-walk-through:-some-places-to-know-about-when-working-with-the-Semantic-File-System)
+    *   [3.1 Semantic Properties Page](#Semantic-Properties-Page)
+    *   [3.2 Team Provider menu](#Team-Provider-menu)
+    *   [3.3 Semantic File System Preference Page](#Semantic-File-System-Preference-Page)
+    *   [3.4 Semantic Content View](#Semantic-Content-View)
+    *   [3.5 Team Synchronizer UI](#Team-Synchronizer-UI)
+    *   [3.6 Remote History Page](#Remote-History-Page)
+*   [4 The DefaultContentProvider](#The-DefaultContentProvider)
+    *   [4.1 Local File handling](#Local-File-handling)
+    *   [4.2 REST resource handlingâInternet URL](#REST-resource-handling.E2.80.94Internet-URL)
+*   [5 Using Semantic Resources outside of Semantic Projects](#Using-Semantic-Resources-outside-of-Semantic-Projects)
+
+Prerequisites
+-------------
+
+You have an Eclipse IDE with the Semantic File System plug-ins running.  
+The required plug-ins are:
+
+*   org.eclipse.core.resources.semantic
+*   org.eclipse.ui.resources.semantic
+
+The IDE used for the below examples was a Helios âEclipse IDE for RCP and RAP Developersâ downloaded from [https://www.eclipse.org/downloads](https://www.eclipse.org/downloads). Other Eclipse IDE 3.5 and 3.6 versions using a 5.0 or 6.0 Java runtime should also work.
+
+Creating a Semantic project
+---------------------------
+
+Before you can start, you need to create a project located in the Semantic File System (also referred to as "Semantic Project"). Since the "Generic Project" wizard offers to do this in the UI, it can be used to do so. The wizard can be started using New -> Project..., then General -> Project:
+
+ ![NewSemanticProjectWizard.jpg](https://raw.githubusercontent.com/https://raw.githubusercontent.com/eclipse-platform/eclipse.platform/docs/master/docs/images/NewSemanticProjectWizard.jpg)
```

Comment:
all the URLs would be better as relative and I think they are **all broken**.
```
images/NewSemanticProjectWizard.jpg
```

Use the preview button to view them.

<img width="224" height="169" alt="image" src="https://github.com/user-attachments/assets/6684fb76-45e1-4568-945b-0a649208f4ea" />